### PR TITLE
kid3, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Not checked on 32bit (yet).

File: media-sound/kid3/patches/kid3-3.9.5.patchset [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+index ddb7afc..c29405f 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -140,6 +140,14 @@ elseif(ANDROID)
+   set(WITH_PLUGINSDIR_DEFAULT ".")
+   set(QT_ANDROID_APP_VERSION ${KID3_VERSION})
+   set(QT_ANDROID_APP_VERSION_CODE 29)
++elseif(HAIKU)
++  set(WITH_DATAROOTDIR_DEFAULT "data")
++  set(WITH_DOCDIR_DEFAULT "data/doc/kid3-qt")
```
Comment (jmairboeck) [inline media-sound/kid3/patches/kid3-3.9.5.patchset:17]: If you are setting it manually anyway, you could set it correctly, i.e. "documentation/kid3-qt" (or "documentation/packages/kid3-qt" if you want to mimic $docDir).

Comment (Begasus) [reply media-sound/kid3/patches/kid3-3.9.5.patchset:17]: I could change it here, patch already merged upstream for this. :)

Comment (Begasus) [reply media-sound/kid3/patches/kid3-3.9.5.patchset]: Updated, seems the documentation isn't installed either way. :)

**Pull request merged**